docs/spec/baseapp/WIP_abci_application.md

```diff
@@ -0,0 +1,46 @@
+# ABCI application
+
+The `BaseApp` struct fulfills the tendermint-abci `Application` interface. 
+
+## Info
+
+## SetOption
+
+## Query
+
+## CheckTx
+
+## InitChain
+TODO pseudo code 
+
+During chain initialization InitChain runs the initialization logic directly on
+the CommitMultiStore and commits it. The deliver and check states are
+initialized with the ChainID. Additionally the Block gas meter is initialized
+with an infinite amount of gas to run any genesis transactions.
+
+Note that we do not `Commit` during `InitChain` however BeginBlock for block 1
```

Comment:
the paragraph above says "and commits it" - I think this line is correct, so what do we mean in the above paragraph?

Reply:
Yeah I was curious about this - these sentences are more or less ripped from code comments - I'm not sure I reckon this is incorrect in both the codebase comment as well as my spec comment here - at least if it is "committing" it the language is not explicit enough - I'll remove from both.